draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1218,568 +1044,686 @@ uCDN-A                         dCDN-B                         dCDN-C
    <vspace blankLines="1"/>
-   Description: A specification of the trigger action and a set of
-         targets upon which to act.
+   Description: Defines the type of the CI/T Trigger Action.
    <vspace blankLines="1"/>
-   Value: A Trigger.v2 Specification, as defined in <xref target="trigger-v2-specification"/>.
+   Value: Trigger Action Type, as defined in <xref target="trigger-action"/>.
    <vspace blankLines="1"/>
-   Mandatory: Yes.
+   Mandatory: Yes. 
    </t>
 
-   <t hangText="Name: cdn-path">
+   <t hangText="Name: specs">
    <vspace blankLines="1"/>
-   Description: The CDN PIDs of CDNs that have already issued the CI/T Command to their dCDNs.
+   Description: Array of trigger specs representing the trigger's targets, as described in <xref target="trigger-specs"/>.
    <vspace blankLines="1"/>
-   Value: A non-empty JSON array of JSON strings, where each string is a CDN PID as defined in 
-   <xref target="loop-detection-and-prevention"/>.
+   Value: Array of GenericTriggerSpec objects (see <xref target="generic-spec-object"/>).
    <vspace blankLines="1"/>
-   Mandatory: No.
-   </t>
-
-   </list></t>
-   </list></t>
-
-   </section>
-
-
-   <section title="CI/T Modify Command" anchor="cit-trigger-modify-command">
-   <t> 
-   CI/T Modify Command, which is used to modify an existing trigger, MUST use a MIME media 
-   type of "application/cdni; ptype=ci-trigger-command.modify.v2".
+   Mandatory: Yes. Furthermore, the list MUST NOT be empty.
    </t>
-   <t>
-      A CI/T Modify Trigger Command is encoded as a JSON object containing the following
-      name/value pairs:</t>
 
-   <t><list style="empty" hangIndent="3">
-   <t><list style="hanging" hangIndent="3"><t hangText="Name: specs">
+   
+   <t hangText="Name: extensions">
    <vspace blankLines="1"/>
-   Description: An updated specification of the trigger action and a set of targets upon which to act.
+   Description: Array of trigger extensions, as described in <xref target="trigger-extensions"/>.
    <vspace blankLines="1"/>
-   Value: A Trigger.v2 Specification, as defined in <xref target="trigger-v2-specification"/>.
+   Value: Array of GenericTriggerExtension objects (see <xref target="generic-extension-object"/>).
    <vspace blankLines="1"/>
-   Mandatory: No, but at least one of "specs" and "desired-status" should be set.
+   Mandatory: No. The default is no extensions.
    </t>
 
-   <t hangText="Name: desired-status">
-   <vspace blankLines="1"/>
-   Description: The target status for an existing trigger as defined in <xref target="trigger-status"/>
+   <t hangText="Name: labels">
    <vspace blankLines="1"/>
-   Value: JSON string. 
+   Description: Array of trigger labels.
    <vspace blankLines="1"/>
-   Mandatory: No, but at least one of "specs" and "desired-status" should be set.
-   </t>
-
-   </list></t>
-   </list></t>
-
-   </section>
+   Value: Array of Trigger labels, Each label is encoded as a JSON string and SHOULD be unique per trigger.
```

Comment:
Punctuation: Convert comma after "labels" to a period.

Reply:
Is there a reason why each label SHOULD be unique per trigger?  It seems like there is value in labels being shared by multiple triggers, right?  Or, do you mean to say that each label within the list should be unique, i.e., appear only once?

Reply:
The latter for sure.
Is there a better wording for this you can think of?

Reply:
Suggested better wording on the new PR